llvm/lib/Target/RISCV/MCTargetDesc/RISCVInstPrinter.cpp

```diff
@@ -158,6 +158,16 @@ void RISCVInstPrinter::printFRMArg(const MCInst *MI, unsigned OpNo,
   O << ", " << RISCVFPRndMode::roundingModeToString(FRMArg);
 }
 
+void RISCVInstPrinter::printFRMArgLegacy(const MCInst *MI, unsigned OpNo,
+                                         const MCSubtargetInfo &STI,
+                                         raw_ostream &O) {
+  auto FRMArg =
+      static_cast<RISCVFPRndMode::RoundingMode>(MI->getOperand(OpNo).getImm());
+  if (PrintAliases && !NoAliases && FRMArg == RISCVFPRndMode::RoundingMode::RNE)
```

Comment:
Should we skip printing RNE always so that no-aliases output can still be parsed by older tools?

Reply:
Good idea, I've made that change. I don't imagine no aliases is commonly used outside of our own backend tests, but no harm in being cautious here.

Reply:
I know people who like to use it with objdump to see compressed instructions with their `c.` prefix or for other reasons.